controller/image.yaml

```diff
@@ -1,20 +1,20 @@
 schema_version: 1
 
 name: "rhpam-7/rhpam-controller-rhel8"
-description: "Red Hat Process Automation Manager Standalone Controller 7.13 container image"
+description: "IBM Business Automation Manager Open Editions Standalone Controller 7.13 container image"
 version: "7.13.0"
 from: "registry.redhat.io/ubi8/ubi-minimal:latest"
 labels:
   - name: "com.redhat.component"
-    value: "rhpam-7-controller-rhel8-container"
+    value: "ibm-bamoe-7-controller-rhel8-container"
```

Comment:
did we got from Comet correct package with vesion 8 ?

Reply:
no